chemprop/uncertainty/calibrator.py

```diff
@@ -1,13 +1,15 @@
 from abc import abstractmethod
 
+from scipy.optimize import fmin
+import torch
 from torch import Tensor
 
 from chemprop.utils.registry import ClassRegistry
 
 
 class UncertaintyCalibrator:
     @abstractmethod
-    def calibrate(self, preds: Tensor, uncs: Tensor, targets: Tensor, mask: Tensor) -> Tensor:
+    def calibrate(self, preds: Tensor, uncs: Tensor, targets: Tensor, mask: Tensor):
```

Comment:
If you want to view these as sklearn-style, then `calibrate` should:
1) either return (a) `Self` or (b) be a `classmethod` that returns a new class (my preference)
2) have `fit` aliased to `calibrate`